### ISSUE TITLE
Cached entries in home tab

### ISSUE THREAD

**Issue by fraluc06** [axiel7/AniHyou-android#185]
When you hold an entry in the home tab you have on watching or other status, it opens a quick menu to modify values about the series. It doesn't reflect the same data as you can see in the anime tab if this specific entry is in your list

**To Reproduce**
1. Go to 'Home tab'
2. Hold on an entry you have on your list but in this tab

**Expected behavior**
it should be like this:
![IMG_20240829_223101](https://github.com/user-attachments/assets/acea39b7-b072-461d-bf1e-a24700b2420d)

Instead I have this:
![IMG_20240829_223047](https://github.com/user-attachments/assets/b2bef534-c228-4170-8903-a28f3a802795)

(Of course I used the same anime)

**Enviroment (required)**
 - Device model: Realms 11 pro 5G (RMX3771) but I think it's not important
 - Android version: 14
 - App version: 1.3.4 (77)

**Comment by axiel7** [comment]
This is because the data is cached and doesn't refresh automatically. For a temporary solution I'll add a swipe to refresh to the Discover tab

**Comment by fraluc06** [comment]
Today, when I opened the app in the Discover tab, I had this behaviour that is the expected behaviour ![Screenshot_2024-08-30-10-50-51-09_65c98e98fed398536bf0a4b3f9f52667.jpg](https://github.com/user-attachments/assets/6753ae65-8201-4a08-9319-04651db15af6)

**Comment by axiel7** [comment]
Yeah of course, as I said the data is cached so to refresh it you need to reopen the app or (in a future version) swipe to refresh
